.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+Please make sure that you are familiar with and follow the Code of Conduct for
+this project (found in the CODE_OF_CONDUCT.md file).
+
+Also, please make sure you're familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+If you're new to contributing to open source projects, you might find this free
+video course helpful: http://kcd.im/pull-request
+
+Please fill out the information below to expedite the review and (hopefully)
+merge of your pull request!
+-->
+
+<!-- What changes are being made? (What feature/bug is being fixed here?) -->
+**What**:
+
+<!-- Why are these changes necessary? -->
+**Why**:
+
+<!-- How were these changes implemented? -->
+**How**:
+
+<!-- Have you done all of these things?  -->
+**Checklist**:
```

Comment:
I actually think we can drop the checklist for this repository, maybe just a part on adding yourself to the contributor table. Thoughts?

Reply:
The checklist can help get a quick glance on the contribution type so I think it's useful; it might need to some polishment tho.

Reply:
Hmm good point, wonder if we can do something better here then. This project doesn't have tests, it's just documentation?

Reply:
Well, I guess the test checkbox should only be for the CLI and the bot repos.

But yeah, since this repo is primarily just documentation, I guess that part can be omitted here but kept/used for the other two.